sdk/core/Azure.Core/src/Cryptography/IKeyEncryptionKey.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Core.Cryptography
+{
+    /// <summary>
+    /// A key which is used to encrypt, or wrap, another key
+    /// </summary>
+    public interface IKeyEncryptionKey
+    {
+        /// <summary>
+        /// Encrypts the specified key using the specified algorithm
+        /// </summary>
+        /// <param name="algorithm">The key wrap algorithm used to encrypt the specified key</param>
+        /// <param name="key">The key to be encrypted</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>The encrypted key bytes</returns>
+        byte[] WrapKey(string algorithm, byte[] key, CancellationToken cancellationToken);
```

Comment:
If clients will be interacting with implementations via the interface, `cancellationToken` should be defaulted so the compiler emits the `default` if otherwise unspecified.